build.gradle

```diff
@@ -86,9 +87,10 @@ distributions {
     }
 }
 
-distTar {
-    archiveName "pulsar-manager.tar"
-}
+//distTar {
```

Comment:
Do you mind giving some info on why this is commented out?

Reply:
Gave me issues as distTar wasn't found in gradle 8.5 and didn't want to deal with it immediately. End goal is of course to have this working again but thought of tackling more important things first.